docs/getting-started-guides/aws/public.md

```diff
@@ -59,6 +59,8 @@ for i in $(ls .terraform/modules/*/Makefile); do i=$(dirname $i); make -C $i; do
 ### Provision the cluster infrastructure
 
 ```
+cd /tmp/kubeform/terraform/aws/public-cloud
```

Comment:
This step is not necessary, we can remove this. We need to add the new terraform version requirement in this file.
For squashing I usually use ```git rebase -i``` you can also use ``` git merge --squash``` or something like https://github.com/tj/git-extras/blob/master/Commands.md#git-squash, or just recreate your local branch as you must have done here,  then you need to ```git push --force``` against your branch when you are sure you want to override it